eyeshade/workers/surveyors.js

```diff
@@ -44,7 +45,7 @@ exports.workers = {
         `
         const votingQ = await client.query(query1, [surveyorId])
         if (!votingQ.rowCount) {
-          throw new Error('no votes for this surveyor!')
+          return runtime.captureException(new Error('no votes for this surveyor!'))
```

Comment:
When do we need to explicitly wrap errors with captureException? My understanding was that sentry would capture these automatically

Reply:
this was when i was debugging. thanks

Reply:
oh, no. this was because the initializer fn that triggers this waits for it to succeed.

Reply:
not sure I entirely understand, do you mind explaining what would happen if we threw here instead of returning early and using captureException?

Reply:
so, here for instance: https://github.com/brave-intl/bat-ledger/blob/master/eyeshade/workers/reports.js#L52 if we have 2 surveyors and the first doesn't have any votes (nothing to do) it will throw instead of just completing the actions for zero items, which causes our second surveyor not to go through the process